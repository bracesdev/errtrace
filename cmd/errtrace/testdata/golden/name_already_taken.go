//go:build ignore

package foo

import "strconv"

func Unwrapped(errtrace string) (int, error) {
	// For some reason, the string is named errtrace.
	// Don't think about it too hard.
	i, err := strconv.Atoi(errtrace)
	return i + 42, err
}
