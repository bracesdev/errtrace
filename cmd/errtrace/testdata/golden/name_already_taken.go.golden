//go:build ignore

package foo

import "strconv"; import errtrace2 "braces.dev/errtrace"

func Unwrapped(errtrace string) (int, error) {
	// For some reason, the string is named errtrace.
	// Don't think about it too hard.
	i, err := strconv.Atoi(errtrace)
	return i + 42, errtrace2.Wrap(err)
}
